nixos/modules/services/x11/desktop-managers/gnome.nix

```diff
@@ -59,7 +59,7 @@ let
     enableGnomePanel = true;
   } ++ cfg.flashback.customSessions;
 
-  notExcluded = pkg: mkDefault (!(lib.elem pkg config.environment.gnome.excludePackages));
+  notExcluded = pkg: mkDefault (!(lib.elem (lib.getName pkg) (map lib.getName config.environment.gnome.excludePackages)));
```

Comment:
Also please keep it in sync with the other desktop environments.

Reply:
Searching for `excludePackages` in the desktop-managers folder, the only other file that has something equivalent to this is cinnamon. I just updated the PR to apply the same fix to that too.